### PR TITLE
New success profile

### PR DESCRIPTION
**Pull request by iryanclarke**
Adds in the new `success` type profile to our three core themes

### CODE REVIEW

File: dist/documentation/themes/dark.js [hunk 1]
```diff
@@ -163,6 +163,59 @@ primary : {textBase:{
             hex: "#1A5740",
             description: "Used for the pressed border colour for primary components such as buttons"
           }},
+success : {textBase:{
+            name: "success.textBase",
+            hex: "#42AD86",
+            description: "Used as the text or fill colour for interactive and static primary components such as links, text, and icons"
```
Comment (alexandra-lim) [inline dist/documentation/themes/dark.js:169]: Let's update all the descriptions to say `success` instead of `primary`!

**Review by alexandra-lim — COMMENTED** [review]
Looking good, just the one comment about updating the descriptions to better suit our new success tokens!

File: dist/documentation/themes/dark.js (hunk 1 above)
Comment (iryanclarke) [reply dist/documentation/themes/dark.js:169]: Updated all of these in `dark`! (Also a few I missed in `light`)

File: dist/documentation/themes/dark.js [hunk 2]
```diff
@@ -163,6 +163,59 @@ primary : {textBase:{
...
+            hex: "#1A5740",
+            description: "Description to come."
+          },backgroundMuted:{
+            name: "success.backgroundMuted",
+            hex: "#13402F",
+            description: "Used as the muted background for successful intention versions of components such as message, pill, or box. Paired with primary.textBase for a subtler contrast"
+          },backgroundMutedHover:{
+            name: "success.backgroundMutedHover",
+            hex: "#1A5740",
+            description: "Used as the hovered muted background for successful intention versions of components such as message, pill, or box. Paired with primary.textBase for a subtler contrast"
```
Comment (CCAyl) [inline dist/documentation/themes/dark.js:197]:
```suggestion
            description: "Used as the hovered muted background for successful intention versions of components such as message, pill, or box. Paired with success.textBase for a subtler contrast"
```

File: dist/documentation/themes/dark.js [hunk 3]
```diff
@@ -163,6 +163,59 @@ primary : {textBase:{
...
+          },backgroundMutedActive:{
+            name: "success.backgroundMutedActive",
+            hex: "#256E53",
+            description: "Used as the pressed muted background for successful intention versions of components such as message, pill, or box. Paired with primary.textBase for a subtler contrast"
```
[the remark this answers is not in the document]
Comment (CCAyl) [reply dist/documentation/themes/dark.js:201]: but we need to change these guys  for all themes!

File: src/data/emerson.json [hunk 4]
```diff
@@ -219,6 +219,79 @@
...
+        },
+        "active": {
+          "value": "{colors.green.800}",
+          "type": "color",
+          "description": "Description to come."
+        },
+        "muted": {
+          "value": "{colors.green.200}",
+          "type": "color",
+          "description": "Used as the muted background for successful intention versions of components such as message, pill, or box. Paired with primary.textBase for a subtler contrast"
```
Comment (alexandra-lim) [inline src/data/emerson.json:259]:
```suggestion
          "description": "Used as the muted background for successful intention versions of components such as message, pill, or box. Paired with success.textBase for a subtler contrast"
```

File: src/data/emerson.json [hunk 5]
```diff
@@ -219,6 +219,79 @@
...
+        },
+        "mutedHover": {
+          "value": "{colors.green.300}",
+          "type": "color",
+          "description": "Used as the hovered muted background for successful intention versions of components such as message, pill, or box. Paired with primary.textBase for a subtler contrast"
```
Comment (alexandra-lim) [inline src/data/emerson.json:264]:
```suggestion
          "description": "Used as the hovered muted background for successful intention versions of components such as message, pill, or box. Paired with success.textBase for a subtler contrast"
```

File: src/data/emerson.json [hunk 6]
```diff
@@ -219,6 +219,79 @@
...
+        },
+        "mutedActive": {
+          "value": "{colors.green.400}",
+          "type": "color",
+          "description": "Used as the pressed muted background for successful intention versions of components such as message, pill, or box. Paired with primary.textBase for a subtler contrast"
```
Comment (alexandra-lim) [inline src/data/emerson.json:269]:
```suggestion
          "description": "Used as the pressed muted background for successful intention versions of components such as message, pill, or box. Paired with success.textBase for a subtler contrast"
```

File: src/data/emerson.json [hunk 7]
```diff
@@ -219,6 +219,79 @@
...
+      "border": {
+        "base": {
+          "value": "{colors.green.600}",
+          "type": "color",
+          "description": "Used for the border colour for successful intention versions of components such as buttons and Message"
+        },
+        "hover": {
+          "value": "{colors.green.700}",
+          "type": "color",
+          "description": "Used for the hovered border colour for primary components such as buttons"
```
Comment (alexandra-lim) [inline src/data/emerson.json:286]:
```suggestion
          "description": "Used for the hovered border colour for successful components such as message"
```

File: src/data/emerson.json [hunk 8]
```diff
@@ -219,6 +219,79 @@
...
+        },
+        "active": {
+          "value": "{colors.green.800}",
+          "type": "color",
+          "description": "Used for the pressed border colour for primary components such as buttons"
```
Comment (alexandra-lim) [inline src/data/emerson.json:291]:
```suggestion
          "description": "Used for the pressed border colour for successful components such as message"
```

File: src/data/dark.json [hunk 9]
```diff
@@ -219,6 +219,79 @@
...
+        },
+        "active": {
+          "value": "{colors.green.300}",
+          "type": "color",
+          "description": "Description to come."
+        },
+        "muted": {
+          "value": "{colors.green.200}",
+          "type": "color",
+          "description": "Used as the muted background for successful intention versions of components such as message, pill, or box. Paired with successful.textBase for a subtler contrast"
```
Comment (alexandra-lim) [inline src/data/dark.json:259]: Should these be `success`?

```suggestion
          "description": "Used as the muted background for successful intention versions of components such as message, pill, or box. Paired with success.textBase for a subtler contrast"
```

File: src/data/dark.json [hunk 10]
```diff
@@ -219,6 +219,79 @@
...
+        },
+        "mutedHover": {
+          "value": "{colors.green.300}",
+          "type": "color",
+          "description": "Used as the hovered muted background for successful intention versions of components such as message, pill, or box. Paired with successful.textBase for a subtler contrast"
```
Comment (alexandra-lim) [inline src/data/dark.json:264]:
```suggestion
          "description": "Used as the hovered muted background for successful intention versions of components such as message, pill, or box. Paired with success.textBase for a subtler contrast"
```

File: src/data/light.json [hunk 11]
```diff
@@ -219,6 +219,79 @@
...
+        },
+        "active": {
+          "value": "{colors.green.800}",
+          "type": "color",
+          "description": "Description to come."
+        },
+        "muted": {
+          "value": "{colors.green.200}",
+          "type": "color",
+          "description": "Used as the muted background for successful intention versions of components such as message, pill, or box. Paired with successful.textBase for a subtler contrast"
```
Comment (alexandra-lim) [inline src/data/light.json:259]:
```suggestion
          "description": "Used as the muted background for successful intention versions of components such as message, pill, or box. Paired with success.textBase for a subtler contrast"
```

File: src/data/light.json [hunk 12]
```diff
@@ -219,6 +219,79 @@
...
+        },
+        "mutedHover": {
+          "value": "{colors.green.300}",
+          "type": "color",
+          "description": "Used as the hovered muted background for successful intention versions of components such as message, pill, or box. Paired with successful.textBase for a subtler contrast"
```
Comment (alexandra-lim) [inline src/data/light.json:264]:
```suggestion
          "description": "Used as the hovered muted background for successful intention versions of components such as message, pill, or box. Paired with success.textBase for a subtler contrast"
```

File: src/data/light.json [hunk 13]
```diff
@@ -219,6 +219,79 @@
...
+        },
+        "mutedActive": {
+          "value": "{colors.green.400}",
+          "type": "color",
+          "description": "Used as the pressed muted background for successful intention versions of components such as message, pill, or box. Paired with successful.textBase for a subtler contrast"
```
Comment (alexandra-lim) [inline src/data/light.json:269]:
```suggestion
          "description": "Used as the pressed muted background for successful intention versions of components such as message, pill, or box. Paired with success.textBase for a subtler contrast"
```

File: src/data/dark.json [hunk 14]
```diff
@@ -219,6 +219,79 @@
...
+        },
+        "mutedActive": {
+          "value": "{colors.green.400}",
+          "type": "color",
+          "description": "Used as the pressed muted background for successful intention versions of components such as message, pill, or box. Paired with successful.textBase for a subtler contrast"
```
Comment (alexandra-lim) [inline src/data/dark.json:269]:
```suggestion
          "description": "Used as the pressed muted background for successful intention versions of components such as message, pill, or box. Paired with success.textBase for a subtler contrast"
```

File: src/data/dark.json (hunk 9 above)
Comment (iryanclarke) [reply src/data/dark.json:259]: Yes they should, although I **just** pushed the update from some edits Cath suggested, it's already reflected to be `success` :)

**Pull request merged**